python/scripts/hpx_environment.py

```diff
@@ -1,6 +1,7 @@
 #! /usr/bin/env python 
 #
 # Copyright (c) 2011 Bryce Lelbach
+# Copyright (c) 2019 Patrick Diehl
 #
 # Distributed under the Boost Software License, Version 1.0. (See accompanying
 # file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
```

Comment:
You should add a `from __future__ import print_function` at the very first statement to this script to ensure compatibility with Python V2 (we don't know what `/usr/env/python` refers to).

Reply:
Actually, I dropped the python 2 support intentionally since python 2.x reaches the end of life in around 8 months (https://pythonclock.org/). However, if we still want to support python 2.x, I can add these.

Reply:
Sure, Python V2 officially ends its live in a couple of month. That does not automatically remove V2 from user's machines for a long time, though. I'd like to see it being held compatible for the time being, not sure how others feel.

Reply:
I'm in favor of dropping Python 2 just for the sake of it being very old. But if importing `__future__` (and the related changes with that) isn't a lot of work then why not. I don't think users will revolt if they will have to install Python 3 though.

Reply:
The import of future has not to be done if we only support python 2.7, since print is defined as a function there to. I tested my changes with python 2.7 on Fedora and all scripts run without any issues. I would prefer to keep them as the are and just say HPX needs ptthon 2.7 or later. I hope none is using python 2.6 any more.